shell/platform/embedder/embedder.cc

```diff
@@ -1292,6 +1292,7 @@ FlutterSemanticsNode CreateEmbedderSemanticsNode(
       node.customAccessibilityActions.size(),
       node.customAccessibilityActions.data(),
       node.platformViewId,
+      node.tooltip.c_str(),
```

Comment:
Could we test this by updating the [Dart fixture](https://github.com/flutter/engine/blob/97974ebb5ff81bcdca12c99183a2ee7054db7d8a/shell/platform/embedder/fixtures/main.dart#L137) and [accessibility test](https://github.com/flutter/engine/blob/97974ebb5ff81bcdca12c99183a2ee7054db7d8a/shell/platform/embedder/tests/embedder_a11y_unittests.cc#L93)?